pydata_sphinx_theme/static/css/blank.css

```diff
@@ -0,0 +1 @@
+/* This file is intentionally left blank. */
```

Comment:
can you provide more detail here about why this is blank? Imagine a new developer in the future that hasn't seen any of these issues coming across the CSS file, can we provide enough context that they understand why this is here?

Reply:
I explained it now.